executor/coprocessor.go

```diff
@@ -50,6 +69,9 @@ func NewCoprocessorDAGHandler(sctx sessionctx.Context) *CoprocessorDAGHandler {
 
 // HandleRequest handles the coprocessor request.
 func (h *CoprocessorDAGHandler) HandleRequest(ctx context.Context, req *coprocessor.Request) *coprocessor.Response {
+	ctx = copHandlerCtx(ctx, req)
+	logutil.Logger(ctx).Debug("handle coprocessor request")
```

Comment:
Please remove those `DEBUG` log, they seem just useful for developing?

Reply:
Yes. I'll remove it.